### DISCUSSION TITLE
Existing XMP face tags should be used

### DISCUSSION THREAD

**Discussion by StefanOltmann** [ente-io/ente#4048] [General]
I just imported by library which has proper face tags on every picture that I assigned with another software. I noticed this metadata is not recognized and the system wants me to yet again tag faces for thousands of images.

I expect the app to use the existing data to get to know „my people“ and use the information to tag any new incoming image.

Aa described in other issues I require face tags to be written back into the XMP so that other tools in my chain can use this information.

**Comment by mjklugman** [comment]
I also would like Ente to recognize existing face regions and names. When Ente provides this capability, I will consider using it. Until then, I'm not going to redo work I have already done.

**Comment by henrivdm** [comment]
I understand that Ente does not want to edit photos as a 'backup service' - but it is a bit contradictory from the perspective of providing some fantastic tools to manage said photos.  If I am going to invest the time to tag my photos through Ente, I WANT those tags to be written into the file metadata, not to mention for Ente to read existing face tags.  

They should at least make writing to file optional.  I also thing they should make a proper desktop 2-way synch optional - that would make it the killer app for photo management across all devices, and in a way that it transferrable.

**Comment by r-cheologist** [comment]
Left a more detailed writeup of this exact issue (with the specific XMP schema digiKam/Lightroom/Capture One use for face regions) over on [#1910](https://github.com/ente-io/ente/discussions/1910), since it covers this plus the related keyword-tag and ratings gaps.
